dashboard/app/views/levels/_match_answer.html.haml

```diff
@@ -1,5 +1,5 @@
 / Show the correct answer to authorized teachers
-- if current_user.try(:authorized_teacher?) && !@script.try(:pd?)
+- if @current_user.try(:should_see_inline_answer?, @script_level)
```

Comment:
Why `current_user` -> `@current_user`?

(Is there any risk that a consumer of this view is rendering it without setting `@current_user`?)